package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -362,7 +363,20 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
-	append_bool "$cfg" ignore "--no-dhcp-interface=$ifname" && return 0
+	append_bool "$cfg" ignore "--no-dhcp-interface=$ifname" && {
+		if [ $ADD_LOCAL_GATENAME -gt 0 ] ; then
+			local dashedname="$( echo $ifname | sed 's/\./-/g' )"
+			local routername="$( uci_get system @system[0] hostname OpenWrt )"
```

Comment:
Its what was there. I didn't like it either, but it wasn't material to my goals. A fixed name or distro name has name collision potential still. Lets smash the MAC in there. **DONE**